interface/wx/arrstr.h

```diff
@@ -403,12 +406,61 @@ int wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2);
 
     See wxDictionaryStringSortAscending() for the dictionary sort description.
 
-    @see wxStringSortDescending()
+    @see wxStringSortDescending(), wxNaturalStringSortAscending()
 
     @since 3.1.0
  */
 int wxDictionaryStringSortAscending(const wxString& s1, const wxString& s2);
 
+
+    /**
+        Case-sensitive comparison used for Natural Sort functions.
+        Functions in the same way as wxDictionaryStringSortAscending(), with
+        the exception that numbers within the string are recognised, and
+        compared numerically, rather than alphabetically. When used for
+        sorting, the result is that e.g. file names containing numbers are
+        sorted in a natural way.
+
+        Comparison is case insensitive.
+        
+        e.g. Sorting using wxDictionaryStringSortAscending() results in:
```

Comment:
Shouldn't some of the strings here start with capital F, to actually demonstrate the difference from the default case-sensitive sorting?

Reply:
The point of the two lists is to illustrate the difference between the normal case-insensitive sort, and the alternative natural sort (which is also case insensitive).  I.E. there's no difference between Dictionary sort and Natural sort W.R.T. case.